### ISSUE TITLE
Garbage font when sv_pure is set to 0

### ISSUE THREAD

**Issue by aviogit** [etlegacy/etlegacy#1864]
Hello there,

I have this problem since... well, forever I think. Whenever I try to `set sv_pure "0"` (to overcome the infamous [`MAX_GAMESTATE_CHARS exceeded`](https://github.com/etlegacy/etlegacy/issues/588) issue), I get garbage characters rendered in my client, making it unusable. Below a few screenshot of the problem.

[Screenshot 1](https://ibb.co/3Nmb1Cr)
[Screenshot 2](https://ibb.co/4sgCLdx)
[Screenshot 3](https://ibb.co/hXKZ886)
[Screenshot 4](https://ibb.co/CBxtrfs)

I'm trying to bisect the issue, cutting away `pk3`s from the server directories (I've already purged the client `.etlegacy` directory, nothing changed).

Anyone have an idea how to debug this problem a little faster? Thanks and merry Christmas to everybody!

P.s. with `sv_pure "1"` everything works beautifully, at least until the `MAX_GAMESTATE_CHARS exceeded` error hits...

**Comment by rafal1137** [comment]
Well its not about `sv_pure` being 0. It is just about having referenced files from server so only those are loaded/used when you are playing on particular server. The MAX_GAMESTATE_CHARS is getting hitted and there are few reasons for it:
* Server having too much maps on the server
* Server having maps with long names
* Server having so many players and maps that `CONFIGSTRING` is getting full/exceeding limit thus throwing this error

**Comment by aviogit** [comment]
Yep, that's the problem but in #588 @ensiform wrote that fixing that issue would be quite complex plus ETLegacy would lose compatibility with older versions of Enemy Territory. And I'm fine with that explanation, but I'd rather not waste so much time keeping my etmain slim... he wrote that `sv_pure 0` could have solved the problem of MAX_GAMESTATE_CHARS exceeded, so I'm making this attempt. Narrowing down the issue, I think etpub is the culprit... any ideas about how to fix this issue?

**Comment by rafal1137** [comment]
> About those screenshots with scrabled text in the Issue.. Didnt vid_restart fix that in past?

As workaround try `vid_restart` to clean the mess with those characters
